_posts/2022-12-24-map-my-village-campaign.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: OSM Ghana Improves Access To Ghanaian Villages With Mapping Party 
```

Comment:
Also use title case: `OSM Ghana improves access to Ghanaian villages with mapping party`